klvdata/future.py

```diff
@@ -67,7 +67,7 @@ class PlatformHeadingAngle(MappedConverterElement):
     # TODO: Design problem or thought needed to better structure converters
     #       to support validation during parsing, getting and setting.
     min_value, max_value, units = 0, 360, 'degrees'
-    min_length, max_length, signed = 2, 2, False
+    min_length, max_length, signed = 2, 2, True

```

Comment:
Is future.py still being used? I thought all the action was happening in misb0601.py and that the tests were setup to import from misb0601.py.

Reply:
Is better remove future.py in my opinion,but i don't want change completly the project structure,Remove this .py?

Reply:
And some conversions don`t work correctly, such as the checksum. I have to fix it